### PR TITLE
Fix for default app template crashing (at least on Erlang R13B03)

### PR DESCRIPTION
**Pull request by capotej**
Couldn't get a demo app running from a fresh new_machine.sh, googling the error found a fix here: https://gist.github.com/878868

(awesome project btw!)

### CODE REVIEW

**Comment by capotej** [comment]
No problem, updated as requested.

**Comment by dylanz** [comment]
Would love to get this in there, ran into the same issue.  Made me look at other frameworks for a little while ;)

File: priv/templates/src/wmskel.app.src [hunk 1]
```diff
@@ -8,6 +8,7 @@
   {applications, [
                   kernel,
                   stdlib,
+		  inets,
```
Comment (Vagabond) [inline priv/templates/src/wmskel.app.src]: Fix the indenting on this line to use spaces, not tabs.

**Comment by Vagabond** [comment]
If you fix the whitespace error noted above, I think this looks good to merge.

**Comment by capotej** [comment]
Fixed, my bad.

**Pull request merged**